### DISCUSSION TITLE
FF7 Crashes loading battles after update to FFNx v.1.16.0.16

### DISCUSSION THREAD

**Discussion by MekonACE** [julianxhokaxhiu/FFNx#595] [General]
Has anyone else come across FF7 crashing when loading into battle with the updated FFNx driver?

![Capture](https://github.com/julianxhokaxhiu/FFNx/assets/144498496/9a1933e5-0309-40e1-a99b-f4a3173b4753)

Any advice would be appreciated!

-Mekon

**Comment by julianxhokaxhiu** [comment]
Please use the Discord channels for support. Thank you.
